### DISCUSSION TITLE
Failure to read JP2 files

### DISCUSSION THREAD

**Discussion by kiloreux** [ImageMagick/ImageMagick#3423] [Help]
I am running identify on this jp2 file. But it's failing with `Aborted` 

Older versions work just fine on the same file. Tried with `7.0.4-10`

Sample file:
https://drive.google.com/file/d/10TaVqWg21Sx4xQkTrDWPmW5ah65x_LYD/view?usp=sharing

**Comment by snibgo** [comment]
Do you mean that 7.0.4-10 succeeded or failed? What was the version number for the failure?

**Reply by kiloreux** [reply]
That version succeeded. The version that failed is the latest.

**Reply by dlemstra** [reply]
I am unable to reproduce your issue with the latest version on Windows that uses openjpeg 2.3.1. Is it possible that you are also using a different version of openjpeg?
